presto-main/src/main/java/io/prestosql/sql/planner/planprinter/WindowOperatorStats.java

```diff
@@ -17,8 +17,6 @@
 import io.prestosql.operator.WindowInfo.DriverWindowInfo;
 import io.prestosql.util.Mergeable;
```

Comment:
commit title is too long. It wraps in github.

Also the below part looks unnecessary (it is obvious from the code). 
```
List of changes:
- removing Precondition from WindowOperatorStats
- documenting the expected behaviour for situation when Stats are created for no DriverWindowInfo
in a test
```

Reply:
I prefer to add this kind of comments even when the changes are simple - just to have the intentions of the change written somewhere.

Reply:
I've made the commit message shorter